DependencyInjection/BernardExtension.php

```diff
@@ -130,4 +134,17 @@ private function registerListeners(array $config, ContainerBuilder $container)
             }
         }
     }
+
+    public function configureProfiler(ContainerBuilder $container, array $config)
+    {
+        if (isset($config['profiler']['enabled']) && !$config['profiler']['enabled']) {
+            return;
+        }
+
+        $definition = new Definition(ProducerCollector::class);
```

Comment:
As the bundle already use xml for configuring services, I think it's better to stick with it if possible. Anyone else have an opinion on this?